src/utils/Placement.ts

```diff
@@ -27,7 +27,7 @@ export class Placement implements PlacementDTO {
     public constructor(dto: PlacementDTO) {
         this.scale = dto.scale;
         this.rotate = dto.rotate;
-        this.translate = dto.translate;
+        this.translate = dto.translate || { x: 0, y: 0 };
```

Comment:
I'll have to check the API, but I think you can usually pass `null` for this value if you don't have anything real to pass.